doc/rst/tools/protoc-gen-chpl/protoc-gen-chpl.rst

```diff
@@ -0,0 +1,186 @@
+===============
+Chapel Protobuf
+===============
+
+Chapel Protobuf is a Google Protocol Buffers implementation for Chapel.
+`Protocol Buffers`_ is a language-neutral, platform-neutral, extensible mechanism
+for serializing structured data. The protocol buffer language is a language for 
+specifying the schema for structured data. This schema is compiled into language
+specific bindings.
+
+This project is comprised of two components:
+
+* Code generator: The ``protoc-gen-chpl`` tool is a compiler plugin to ``protoc``, the protocol
+  buffer compiler. It provides arguments to the ``protoc`` compiler so that it knows how to
+  generate Chapel specific code for a given ``.proto`` file.
+  
+* Chapel library: These module has the runtime implementation of protobufs
+  in Chapel and provides functionality to serialize messages in wire format.
+
+
+Installation Instructions
+=========================
+
+In ``$CHPL_HOME`` run the following:
```

Comment:
This should start out by saying something about needing protobuf installed. Since you need header files, you probably need the `-dev` package on debian/ubuntu. If you have the `apt` commands handy it would be nice to show them like, "For example, to install protobuf on an Ubuntu system, you might use ``apt-get install libprotobuf-dev``". You don't have to cover all systems here but having some concrete detail can be helpful.

Reply:
> For example, to install protobuf on an Ubuntu system, you might use `apt-get install libprotobuf-dev`

I would rather prefer giving a link to the protobuf compiler official installation [guide](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation). If that's ok?

Reply:
Those instructions don't seem to indicate that it is common for linux distributions to have packages for it, so I'd still put the part about `apt-get`.

Reply:
Sure!